include/swift/Option/Options.td

```diff
@@ -1336,9 +1336,9 @@ def dump_scope_maps : Separate<["-"], "dump-scope-maps">,
   MetaVarName<"<expanded-or-list-of-line:column>">,
   ModeOpt,
   Flags<[FrontendOption, NoInteractiveOption, DoesNotAffectIncrementalBuild]>;
-def dump_type_refinement_contexts :
```

Comment:
Please also update `Options.swift` in [swift-driver](https://github.com/swiftlang/swift-driver).

Reply:
https://github.com/swiftlang/swift-driver/pull/1728